packages/components/ng-shell-tracking/src/provider.js

```diff
@@ -4,7 +4,7 @@ export default class NgAtInternet {
   }
 
   $get() {
-    this.trackingPlugin.init();
+    console.log(this.trackingPlugin);
```

Comment:
to remove

Reply:
Yes, it's here just because i needed to debug on PF's